fixes and update of readme for v0.9 release

rsnancollas opened the pull request:
Made a few changes to support the 0.9 release

rsnancollas committed:
fixes and update of readme for v0.9 release

asadate commented on support-docs/README.md:
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
+|maxTemperature|maximum temperature under which a resistor can be expected to reliably operate|definitions.json#/unit| |
+|resistorDerating|graph object to capture resistance changes with temperature|definitions.json#/graphDefiniton| |
+|package|package size of resistor|graph.json#/package| |
+
```
type: "definitions.json#/package", not "graph.json..."

rsnancollas replied:
Looks fixed.

asadate commented on part-spec/graph.json:
```diff
@@ -1,61 +1,65 @@
-                "type": "array",
-                "items": {
+    "$id": "https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/Registers.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
```
"/part-spec/graph.json" not "Registers.json"

rsnancollas replied:
Looks fixed.

asadate commented on support-docs/README.md:
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
+|partModelInformation|reference to an external XML file that contains a part model in IPC/DAC2552 format|#/$definitions/externalFile| |
+|standardPackageSize|name of standard package size (imperial)|Must set either Ref or Type| |
+|standardPackageType|name of standard package types|Must set either Ref or Type| |
+
```
standardPackageType & standardPackageSize: "do we want to add string as a type"?

rsnancollas replied:
good call.

asadate commented on support-docs/README.md:
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
+|typValue|typical value|Number| |
+|minValue|minimum value|Number| |
+|maxValue|maximum value|Number| |
+
```
It looks like we forgot to remove "value" when we merged it with "unit".

rsnancollas replied:
I didn't make this change in this version - that's for 1.0, so I think it's okay we don't have this fixed yet.

asadate commented on support-docs/README.md:
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
+|temperatureCoefficient|change in capacitance when the temperature is changed|Number| |
+|minTemperature|minimum temperature under which a capacitor can be expected to reliably operate|definitions.json#/unit| |
+|maxTemperature|maximum temperature under which a capacitor can be expected to reliably operate|definitions.json#/unit| |
+|capacitorDerating|graph object to capture capacitance changes with voltage|definitions.json#/graphDefiniton| |
```
Should be"graph.json#/graphDefinition".

rsnancollas replied:
looks fixed.

asadate commented on support-docs/README.md:
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
+|temperatureCoefficient|change in inductance when the temperature is changed|Number| |
+|minTemperature|minimum temperature under which a inductor can be expected to reliably operate|definitions.json#/unit| |
+|maxTemperature|maximum temperature under which a inductor can be expected to reliably operate|definitions.json#/unit| |
+|saturationCurve|graph object to capture inductor saturation with current|definitions.json#/graphDefiniton| |
```
graph.json#

rsnancollas replied:
thanks for fixing.

asadate commented on support-docs/README.md:
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
+|minTemperature|minimum temperature under which a inductor can be expected to reliably operate|definitions.json#/unit| |
+|maxTemperature|maximum temperature under which a inductor can be expected to reliably operate|definitions.json#/unit| |
+|saturationCurve|graph object to capture inductor saturation with current|definitions.json#/graphDefiniton| |
+|resonantFrequencyCurve|graph object to capture inductor resonant frequency|definitions.json#/graphDefiniton| |
```
Should be "graph.json#/graphDefinition".

rsnancollas replied:
looks done.

asadate commented on support-docs/README.md:
```diff
@@ -47,283 +47,374 @@ A digital datasheet shall include the following information:
+|:----|:----|:----|:----|
+|componentID|common component identifying information, such as mpn|definitions.json#/componentID|Yes|
+|diffModeCutoff|frequency at which the differential mode attenuation equals -3dB|definitions.json#/unit| |
+|commonModeAttenuation|graph object to capture common mode attenuation of a common mode choke at various frequencies|definitions.json#/graphDefiniton| |
```
"graph.json#/graphDefinition". It looks like we have to make these  fixes in the source files (passive.json, power.json,..)

rsnancollas replied:
thanks for fixing this. I'll re-generate.

asadate reviewed (changes requested):
Looks like the script is working well. Except for 1 or 2 issues, most of the issues I've flagged can be traced to the source file. Mostly around the location of "graphDefinition".  Let's fix the issues in the power.json and passive.json files before merging this version

asadate committed:
Update graph.json

changed $id name

asadate committed:
Update passives.json

Made changes to fix reference file error "definitions.json#/graphDefinition" to "grap,json#/graphDefinition"

asadate committed:
Update power.json

Made changes to fix references from "definitions.json#/graphDefinition" to "graph.json#/graphDefinition"

rsnancollas committed:
changes to fix units and readme for v0.9 release

rsnancollas commented:
RE-ran the script and updated the README.
